client/pokemon/pokemon.ctrl.js

```diff
@@ -59,6 +59,7 @@ module.exports = class Pokemon {
     this.paddedTid = (this.data.idNo || this.data.tid).toString().padStart(tidSize, '0');
     this.paddedEsv = this.data.esv.toString().padStart(4, '0');
     this.parsedNickname = replace3dsUnicodeChars(this.data.nickname);
+    this.dispGender = this.data.dexNo !== 29 && this.data.dexNo !== 32 ? this.data.gender : '';
```

Comment:
Can you add a comment here explaining this exception? Without context it seems like it would be difficult to figure out what's going on.